Xamarin.Essentials/Xamarin.Essentials.csproj

```diff
@@ -106,6 +106,22 @@
     <Compile Include="**\*.tizen.cs" />
     <Compile Include="**\*.tizen.*.cs" />
   </ItemGroup>
+  <ItemGroup Condition=" $(TargetFramework.StartsWith('net461')) ">
+    <Compile Include="**\*.win32.*.cs" />
+    <Compile Include="**\*.win32.cs" />
+    <PackageReference Include="System.ValueTuple" Version="4.5.0" />
+    <Reference Include="System.Management" />
+    <Reference Include="System.Speech" />
+    <Reference Include="System.Web" />
+    <Reference Include="System.Windows.Forms" />
+    <Reference Include="System.Xaml">
+      <RequiredTargetFramework>4.0</RequiredTargetFramework>
+    </Reference>
+    <Reference Include="WindowsBase" />
+    <Reference Include="PresentationCore" />
+    <Reference Include="PresentationFramework" />
```

Comment:
Just checking to see if we actually use the presentation bits for this project? We shouldn't be using any of the Xaml-related things. Also, this _should_ work for winforms without this?

Reply:
At the moment it uses WPF dependencies for MainThread, to support either WinForms or WPF will require some reworking but this is a sensible point. I will investigate alternative methods of identifying the thread.

Reply:
Taking a dependency on Newtonsoft.Json in Essentials will make it unusable in some platforms such as WatchOS because the AOT generates a massive amount of code https://github.com/xamarin/xamarin-macios/issues/7250

Since only a limited number of types are supported in Preferences (bool, double, int, float, long, string, and DateTime) could you manually convert them to/from strings?